packages/runtimes/js/src/bundle-url.js

```diff
@@ -37,6 +37,9 @@ function getBaseURL(url) {
 
 // TODO: Replace uses with `new URL(url).origin` when ie11 is no longer supported.
```

Comment:
Do we still need to support IE11 or can we polyfill `URL` dynamically if the user requests IE11 support? Seems it would simplify this code a lot. (potential polyfill https://yarnpkg.com/package/url-polyfill)

cc @mischnic

Reply:
Every time we included `let` or some newer API like `"...".includes` in a runtime/the prelude, someone opened an issue that it doesn't work on IE11, so I guess we should still support it.

And in this case, `getOrigin()` is 170 bytes, while that polyfill is 6.2kb (both minified but not gzip).

Reply:
Right, so there's still more need for IE11 than I thought. Thanks for clearing that up.

The polyfill was just a suggestion and should've probably been inserted only if there's IE11 in browserslist, so the bundle would probably be smaller on newer browsers as we don't do any kind of polyfill and always used built-in URL